lib/services/computeManagement2/package.json

```diff
@@ -12,7 +12,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Sun, Carissa <caris@microsoft.com>"
   ],
-  "version": "1.0.0-preview",
+  "version": "2.0.0-preview",
```

Comment:
The API version is upgraded from ```preview``` to non-```preview```, and there is one property field that is deprecated. Does it qualify?